src/app/app-routing.module.ts

```diff
@@ -0,0 +1,35 @@
+import { NgModule } from "@angular/core";
+import { Routes, RouterModule } from "@angular/router";
+
+// components
+import { NotFoundComponent } from "./not-found/not-found.component";
+
+const routes: Routes = [
+    {
+        path: "users",
+        loadChildren: "./users/users.module#UsersModule"
+    },
+    {
+        path: "",
```

Comment:
move it to the top because it is app entry point
```
+    {
 +        path: "",
 +        pathMatch: "full",
 +        redirectTo: "/users/my-account"
 +    },
```